### PR TITLE
Adding Activity1 for JSappy bird

### PR DESCRIPTION
**Pull request by sasyak**

### CODE REVIEW

File: content/english/jsappybird/1-Activity/Change-screen-text.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Change the Screen text"
+date: 2020-02-10T13:24:17-07:00
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Change-screen-text.md:3]: Change the date to today's date

File: content/english/jsappybird/1-Activity/Change-screen-text.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+draft: false
+weight: 5
+---
+
+Lets change the start screen text
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Change-screen-text.md:8]:
```suggestion
Let's change the start screen text:
```

File: content/english/jsappybird/1-Activity/Change-screen-text.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+     Add below command in the file [File: start-screen.js]
+     var text = game.add.text(0, 0, "Press Space to Start", textOptions);
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Change-screen-text.md:11]:
```suggestion
Add below command in the file [File: start-screen.js]
     var text = game.add.text(0, 0, "Press Space to Start", textOptions);
```

File: content/english/jsappybird/1-Activity/Change-screen-text.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `var text = game.add.text(0, 0, "Press Space to Start", textOptions);`.
+
+![alt text](../../img/startscreen.png "image to add the bird in the file")
+
+ In  your console you should see Press Space to Start after you press **run**:
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Change-screen-text.md:20]:
```suggestion
In your console you should see `Press Space to Start` after you press **run**:
```

File: content/english/jsappybird/1-Activity/Change-screen-text.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+![alt text](../../img/startscreen_output.png "brid image in the output")
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Change-screen-text.md:22]:
```suggestion
![alt text](../../img/startscreen_output.png "bird image in the output")
```

File: content/english/jsappybird/1-Activity/Change-screen-text.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+
+
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Change-screen-text.md:25]: Please remove extra lines

File: content/english/jsappybird/1-Activity/Gravity.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+title: "Add Gravity"
+date: 2020-02-10T13:24:17-07:00
+draft: false
+weight: 7
+---
+
+Lets add Gravity to the Bird
+
+    Add below command in the file [File: create-game.js]
+    bird.body.gravity.y= 900;
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Gravity.md:11]:
```suggestion
Let's add gravity to the bird:

Add below command in the file [File: create-game.js]
    bird.body.gravity.y = 900;
```

File: content/english/jsappybird/1-Activity/Gravity.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `bird.body.gravity.y= 900;`.
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Gravity.md:16]:
```suggestion
In the repl.it window below, we started the code with the line `bird.body.gravity.y = 900;`.
```

File: content/english/jsappybird/1-Activity/Gravity.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+![alt text](../../img/gravity.png "image to add gravity to the bird")
+
+ In  your console you should see a Jsappy bird  jumping with gravity after you press **run**:
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Gravity.md:20]:
```suggestion
In your console you should see a JSappy bird jumping with gravity after you press **run**:
```

File: content/english/jsappybird/1-Activity/Steps-to-start.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Steps to start Scafflod"
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Steps-to-start.md:2]:
```suggestion
title: "Starting up"
```

File: content/english/jsappybird/repl-it/_index.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Starting up with Repl.it"
+date: 2020-07-28T13:24:17-07:00
+draft: false
+weight: 1
+---
+
+Today you will be using Repl.it to build JSappy bird. Here is an example interactive window that you will use today:
+
+Open your favorite internet browser (such as Edge, Firefox, Google Chrome, Internet Explorer) and go to https://repl.it.
```
Comment (ozhang22) [inline content/english/jsappybird/repl-it/_index.md:10]:
```suggestion
Open your favorite internet browser (such as Edge, Firefox, Google Chrome, Internet Explorer) and go to [https://repl.it](https://repl.it).
```

File: content/english/jsappybird/1-Activity/jump.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Make the Bird Jump"
+date: 2020-02-10T13:24:17-07:00
+draft: false
+weight: 6
+---
+
+What should we do with the bird !!!!
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump.md:8]:
```suggestion
What should we do with the bird?
```

File: content/english/jsappybird/1-Activity/jump.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+Lets make him Jump
+
+     Add below command in the file [File: create-game.js]
+     spaceKey.onDown.add(jump, this);
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump.md:13]:
```suggestion
Let's make him jump:

Add below command in the file [File: create-game.js]
     spaceKey.onDown.add(jump, this);
```

File: content/english/jsappybird/1-Activity/jump.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `spaceKey.onDown.add(jump, this);`.
+
+![alt text](../../img/jump.png "image to add jump down")
+
+ In  your console you should see a Jsappy bird jumping after you press **run**:
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump.md:22]:
```suggestion
In your console you should see a Jsappy bird jumping after you press **run**:
```

File: content/english/jsappybird/1-Activity/jump-force.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+![alt text](../../img/jump_output.png "bird jumping with gravity")
+
+
+
+### Bonus Activity
+
+Lets make the Bird jump up and forwards
+
+    Add below command in the file [File: create-game.js]
+    bird.body.velocity.x= 100;
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump-force.md:31]:
```suggestion
Let's make the bird jump up and forwards:

Add below command in the file [File: create-game.js]
    bird.body.velocity.x = 100;
```

File: content/english/jsappybird/1-Activity/jump-force.md [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+
+Lets add Force to the Bird
+
+    Add below command in the file [File: create-game.js]
+    bird.body.velocity.y= -350;
+
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `bird.body.velocity.y= -350;`.
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump-force.md:16]:
```suggestion
In the repl.it window below, we started the code with the line `bird.body.velocity.y = -350;`.
```

File: content/english/jsappybird/1-Activity/jump-force.md [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+
+![alt text](../../img/jump_force.png "image to add gravity to the bird")
+
+ In  your console you should see a Jsappy bird  jumping with force after you press **run**:
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump-force.md:20]:
```suggestion
In your console you should see a Jsappy bird jumping with force after you press **run**:
```

File: content/english/jsappybird/1-Activity/jump-force.md [hunk 18]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Make the Bird Jump with Force"
+date: 2020-07-29T13:24:17-07:00
+draft: false
+weight: 8
+---
+
+Lets add Force to the Bird
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump-force.md:8]:
```suggestion
Let's add force to the bird
```

File: content/english/jsappybird/1-Activity/jump-force.md [hunk 19]
```diff
@@ -0,0 +1,34 @@
...
+
+    Add below command in the file [File: create-game.js]
+    bird.body.velocity.y= -350;
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/jump-force.md:11]:
```suggestion
Add the below command in the file [File: create-game.js]
    bird.body.velocity.y = -350;
```

File: content/english/jsappybird/1-Activity/add-rotation.md [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+draft: false
+weight: 9
+---
+
+Lets add Force to the Bird
+
+    Add below command in the file [File: update-game.js]
+    if(bird.angle<50) {
+    bird.angle+=1;
+    }
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-rotation.md:13]:
```suggestion
Let's add rotation to the bird:

Add below command in the file [File: update-game.js]
    if (bird.angle < 50) {
       bird.angle += 1;
    }
```

File: content/english/jsappybird/1-Activity/add-rotation.md [hunk 21]
```diff
@@ -0,0 +1,34 @@
...
+
+## Comments:
+
+If is a conditonal block where we are checking the angle of the bird and if thta is less than 50 we are exceuting the condition
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-rotation.md:17]:
```suggestion
`If` is a conditional block where we are checking the angle of the bird and if it's less than 50 we are executing the condition.
```

File: content/english/jsappybird/1-Activity/add-rotation.md [hunk 22]
```diff
@@ -0,0 +1,34 @@
...
+
+    Add below command in the file [File: update-game.js]
+    if(bird.angle<50) {
+    bird.angle+=1;
+    }
+
+## Comments:
+
+If is a conditonal block where we are checking the angle of the bird and if thta is less than 50 we are exceuting the condition 
+   -bird.angle+=1;
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-rotation.md:18]: A little confused about this.

File: content/english/jsappybird/1-Activity/add-rotation.md [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `bird.angle+=1;`.
+
+![alt text](../../img/rotate.png "image to add rotation to the bird")
+
+ In  your console you should see a Jsappy bird  roatting with an angle after you press **run**:
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-rotation.md:26]:
```suggestion
In the repl.it window below, we started the code with the line `bird.angle +=1;`.

![alt text](../../img/rotate.png "image to add rotation to the bird")

In your console you should see a Jsappy bird rotating with an angle after you press **run**:
```

File: content/english/jsappybird/1-Activity/add-jump-sound.md [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+title: "Add Sound While the Bird is Jumping"
+date: 2020-07-29T13:24:17-07:00
+draft: false
+weight: 10
+---
+
+Lets add Force to the Bird
+
+    Add below command in the file [File: create-game.js]
+    jumpSound.play();
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-jump-sound.md:11]:
```suggestion
Let's add sounds to the bird while it is jumping

Add the below command in the file [File: create-game.js]
    jumpSound.play();
```

File: content/english/jsappybird/1-Activity/add-jump-sound.md [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `jumpSound.play();`.
+
+![alt text](../../img/jump_sound.png "image to add sound to the bird")
+
+ In  your console you should see a Jsappy bird  jumping with sound after you press **run**:
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-jump-sound.md:20]:
```suggestion
In your console you should see a Jsappy bird jumping with sound after you press **run**:
```

File: content/english/jsappybird/1-Activity/_index.md [hunk 26]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Activity 1"
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/_index.md:2]: What does this file do?

File: content/english/jsappybird/1-Activity/load-the-bird.md [hunk 27]
```diff
@@ -0,0 +1,26 @@
...
+title: "Load the bird image"
+date: 2020-02-10T13:24:17-07:00
+draft: false
+weight: 4
+---
+
+First of all Load Flappy and put on the screen.
+
+       Add below command in the file [File: load-sprites.js]
+       game.load.image('bird','assets/bird.png');
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/load-the-bird.md:11]:
```suggestion
First of all let's load Flappy and put it on the screen.

Add below command in the file [File: load-sprites.js]
       game.load.image('bird','assets/bird.png');
```

File: content/english/jsappybird/1-Activity/load-the-bird.md [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `game.load.image('bird', 'assets/bird.png');`.
+
+![alt text](../../img/loadbird.png "image to add the bird in the file")
+
+ In  your console you should see a Jsappy bird after you press **run**:
+
+![alt text](../../img/loadbird_output.png "brid image in the output")
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/load-the-bird.md:22]:
```suggestion
In your console you should see a Jsappy bird after you press **run**:

![alt text](../../img/loadbird_output.png "bird image in the output")
```

File: content/english/jsappybird/1-Activity/Change-screen-text.md (hunk 1 above)
Comment (sasyak) [reply content/english/jsappybird/1-Activity/Change-screen-text.md:3]: Updated the date

File: content/english/jsappybird/1-Activity/Change-screen-text.md (hunk 6 above)
Comment (sasyak) [reply content/english/jsappybird/1-Activity/Change-screen-text.md]: Done

File: content/english/jsappybird/1-Activity/_index.md (hunk 26 above)
Comment (sasyak) [reply content/english/jsappybird/1-Activity/_index.md:2]: this is more like Index file which has all the items for Activity 1

File: content/english/jsappybird/1-Activity/Steps-to-start.md [hunk 29]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Starting up"
+date: 2020-02-10T13:24:17-07:00
+draft: false
+weight: 3
+---
+
+
+Open the Activity-1 Scafflod URL to do the activity.
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Steps-to-start.md:9]:
```suggestion
Open the Activity-1 Scaffold URL to do the activity.
```

File: content/english/jsappybird/1-Activity/Steps-to-start.md [hunk 30]
```diff
@@ -0,0 +1,17 @@
...
+https://repl.it/@nuevofoundation/JSappyBird-Activity-1-Scaffold
+
+When you make changes to the scafflod you will be created as anonymous users.
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/Steps-to-start.md:12]:
```suggestion
When you make changes to the scaffold you will be created as anonymous users.
```

File: content/english/jsappybird/1-Activity/add-jump-sound.md [hunk 31]
```diff
@@ -0,0 +1,26 @@
...
+
+## Working Together
+
+In the repl.it window below, we started the code with the line `jumpSound.play();`.
+
+![alt text](../../img/jump_sound.png "image to add sound to the bird")
+
+In your console you should see a Jsappy bird jumping with sound after you press **run**:
+
+** Bird making Jump sound while its Jumping**
```
Comment (ozhang22) [inline content/english/jsappybird/1-Activity/add-jump-sound.md:22]: What's this line doing?

File: content/english/jsappybird/1-Activity/add-rotation.md (hunk 22 above)
Comment (sasyak) [reply content/english/jsappybird/1-Activity/add-rotation.md:18]: This was part of the original code.

File: content/english/jsappybird/1-Activity/add-jump-sound.md (hunk 31 above)
Comment (sasyak) [reply content/english/jsappybird/1-Activity/add-jump-sound.md:22]: this was a comment ,as I cannot show output with sound.
I will update to reflect as comment

**Pull request merged**